assets/fma-template.md

```diff
@@ -42,60 +42,64 @@
 > - Include both implementation and operational failure modes
 > - Provide references to support the reviewer.
 > - The size of the document will likely be proportional to the project's complexity.
-> - The ultimate goal of this document is to identify action items to improve the security of the  project. The FMA review process can be accelerated by proactively identifying action items during the writing process.
+> - The ultimate goal of this document is to identify action items to improve the security of the project. The FMA review process can be accelerated by proactively identifying action items during the writing process.
 
 ## Introduction
 
-This document covers *[project name, high-level summary of the project, and scope of this analysis].*
+This document covers _[project name, high-level summary of the project, and scope of this analysis]._
 
 Below are references for this project:
 
-- *Link 1, e.g. project charter or design doc*
-- *Link 2, etc.*
+- _Link 1, e.g. project charter or design doc_
+- _Link 2, etc._
 
 ## Failure Modes and Recovery Paths
 
-***Use one sub-header per failure mode, so the full set of failure modes is easily scannable from the table of contents.***
+**_Use one sub-header per failure mode, so the full set of failure modes is easily scannable from the table of contents._**
 
-### [Name of Failure Mode 1]
+### FM1: [Name of Failure Mode 1]
 
-- **Description:** *Details of the failure mode go here. What the causes and effects of this failure?*
-- **Risk Assessment:** *Simple low/medium/high rating of impact (severity) + likelihood.*
-- **Mitigations:** *What mitigations are in place, or what should we add, to reduce the chance of this occurring?*
-- **Detection:** *How do we detect if this occurs?*
-- **Recovery Path(s)**: *How do we resolve this? Is it a simple, quick recovery or a big effort? Would recovery require a governance vote or a hard fork?*
+- **Description:** _Details of the failure mode go here. What the causes and effects of this failure?_
+- **Risk Assessment:** _Simple low/medium/high rating of impact (severity) + likelihood._
+- **Prevention and Mitigations:** _What mechanisms are in place, or what should we add, to:_
+  _1. reduce the chance of this occurring?_
+  _2. reduce the impact of this occurring?_
```

Comment:
Update this section to specifically call out prevention along side impact mitigation. Open to different terminology.

Reply:
What if we just change from:

```diff
- **Mitigations:** *What mitigations are in place, or what should we add, to reduce the chance of this occurring?*
+ **Mitigations:** *What mitigations are in place, or what should we add, to reduce the chance of this occurring and to reduce the impact if it occurs?*
```
I don't want to overconstrain what "mitigation" means by splitting it up into prevention and mitigation, since the words have similar meanings and historically we've treated mitigation as prevention here

Reply:
Sure. The main thing to me is that we are thinking about both prevention and impact reduction. To that end, wdyt about having the heading as just `**Mitigations**`, but also having the two items listed explicitly as in the PR diff?

Reply:
I'm not sure I fully understand the suggestion, perhaps you could push the change to demonstrate?

Reply:
Just this: b243b54a9f6e